km62/Antonov_Mykhailo/laboratory1.sql

```diff
@@ -1,2 +1,126 @@
 -- LABORATORY WORK 1
 -- BY Antonov_Mykhailo
+
+/*---------------------------------------------------------------------------
+1.Створити схему даних з назвою – прізвище студента, у якій користувач може: 
+модифікувати таблиці та вставляти дані
+4 бали
+
+---------------------------------------------------------------------------*/
+--Код відповідь:
+
+
+CREATE USER antonov IDENTIFIED by antonov
+DEFAULT TABLESPACE "SYSTEM"
+TEMPORARY TABLESPACE "TEMP";
+
+GRANT "CONNECT" TO antonov ;
+GRANT "DATAPUMP_IMP_FULL_DATABASE" TO antonov ;
```

Comment:
You had system privileges added by Role. In the task you were asked to add concrete system privilege by GRANT. NOT ROLE. It might be better to use 'INSERT' and 'ALTER'. Please, read your task again. You have done this task, but your user will have more system privileges than he was allowed.